hoomd/mpcd/pytest/test_collide.py

```diff
@@ -338,7 +354,7 @@ def test_rigid_collide_free(
         # create simulation
         total_mass = properties_rigid["mass"][0]
         initial_snap = two_particle_snapshot_factory(
-            particle_types=["A", "B", "C", "D"], L=21
+            particle_types=["A", "B", "C", "D"], L=11
```

Comment:
Please document the reason for these changes with a comment, and submit in a separate small PR.